src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -225,25 +223,26 @@ namespace OpenMS
 
     /**
      * @brief encapsulates readSpectra with file loading convenience if only a file path is given
+     *
      * @param spectra_file file to the SpectrumContainer, must be valid path to mz file
      */
-    void readMzFile(const String& spectra_file)
+    void readMzFileMetaData(const String& spectra_file)
```

Comment:
@mwalzer: I think that is the only use case. Unless I really overlooked something, `readMzFileMetaData`, `setSpectraDataRef` and the `spectra_data_ref` member can all be removed. "spectra_data" is set in `addMissingSpectrumReferences` based on the filename that is passed to that function.